### DISCUSSION TITLE
怎样让任意用户都能访问同一个公开文件？

### DISCUSSION THREAD

**Discussion by Gokou-Ruri** [AlistGo/alist#6014] [Q&A]
假设有各自独立的目录A、B、C，用户1能访问A，用户2能访问B，而我现在想把目录C设置成公开目录，任何人（无论有没有账户，无论账户权限是什么）都能通过网页或webdav访问C，需要怎么操作？我试过给来宾账户访问C，但用户1和2依旧不能访问C，必须退出账号才能访问C。

**Comment by pongfcnkl** [comment]
把c用别名放到a和b的目录下
https://alist.nn.ci/zh/guide/drivers/alias.html

**Reply by Gokou-Ruri** [reply]
这个思路真不错，谢谢。
